server/atl_requests/__init__.py

```diff
@@ -0,0 +1,58 @@
+from server.models import Authentication
+import atlassian_jwt
+import requests
+from urllib.parse import urlencode
+from server.atl_requests.lib import AtLib
+
+
+class Ctx:
+    def __init__(self, org):
+        self.tenant_info = Authentication.query.filter_by(baseUrl=org).first()
+        self.base_url = self.tenant_info.baseUrl
+        self.key = self.tenant_info.key
+        self.shared_secret = self.tenant_info.sharedSecret
+
+
+class AtlassianRequest:
```

Comment:
This is more of an `AtlassianRequestor` since you'll only have the single instance of it